react/features/large-video/components/LargeVideo.web.js

```diff
@@ -71,7 +90,9 @@ class LargeVideo extends Component<Props> {
 
                 <Watermarks />
 
-                <div id = 'dominantSpeaker'>
+                <div
+                    id = 'dominantSpeaker'
+                    onTouchEnd = { this._onDoubleTap }>
```

Comment:
I think checking double-tap by measuring interval between 2 onTouchStart being very small would be much safer for this, as onTouchEnd can be triggered following a long touch and not a tap, which might have a different meaning/handling. so basically with the current approach we might have a long touch followed by a quick tap considered as double tap. WDYT?

Reply:
actually with my proposal we don t need need the onToucheEnd at all, as since the second tap comes quickly after the first, it is unlekly that the user first tapped then long pressed.

Reply:
IMHO this is just a question of probability? Which one is more probable: lang press first and rapid tap after or rapid tap first and long press after? I don't think we have data about that.

Also I haven't used `onTouchEnd` instead of `onTouchStart` because onTouchStart does not work. There is a handler attached somewhere that reverses the tile view change (you can test to see it)